lib/stripe.rb

```diff
@@ -103,7 +103,7 @@ module Stripe
 
   @api_base = "https://api.stripe.com"
   @connect_base = "https://connect.stripe.com"
-  @uploads_base = "https://uploads.stripe.com"
+  @uploads_base = "https://files.stripe.com"
```

Comment:
We may eventually want to rename this to `files_base` for consistency.

Reply:
Yeah, I thought of that but was afraid it should count as a breaking change. wdyt?

Reply:
Ah, true (although hopefully no one's overriding this). It does seem less churnful to just leave it for now though, so NM. If we ever feel strongly about it later, we can always change it then.